geopandas/tests/test_geodataframe.py

```diff
@@ -55,16 +55,12 @@ def test_different_geo_colname(self):
 
         geom2 = [Point(x, y) for x, y in zip(range(5, 10), range(5))]
         df2 = df.set_geometry(geom2, crs='dummy_crs')
-        self.assert_('geometry' in df2)
```

Comment:
Removed this assertion, as there should be no `geometry` column if using a different name.